src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml

```diff
@@ -62,7 +62,7 @@ limitations under the License.
       <arg name="IdentifyTime" type="int16u"/>
       <access op="invoke" privilege="manage"/>
     </command>
-    <command source="client" code="0x40" name="TriggerEffect" optional="true">
+    <command source="client" code="0x40" name="TriggerEffect" optional="false">
```

Comment:
This is _an optional command_ in the spec.  Why is this being changed to not follow the spec?

Reply:
This should be reverted and that should also revert the above TriggerEffect code change